SYCL/Assert/assert_in_kernels.cpp

```diff
@@ -16,7 +16,7 @@
 // CHECK-NOT:  test aborts earlier, one shouldn't see this message
 // CHECK-NOT:  The test ended.
 //
-// CHECK-ACC-NOT: {{.*}}assert_in_kernels.hpp:26: void kernelFunc2(int *, int): {{.*}} [{{[0,2]}},0,0], {{.*}} [0,0,0]
+// CHECK-ACC-NOT: {{.*}}assert_in_kernels.hpp:25: void kernelFunc2(int *, int): {{.*}} [{{[0,2]}},0,0], {{.*}} [0,0,0]
```

Comment:
Do we really  need to check the line number here? Can we not have {{.*}}? Thanks

Reply:
Well, I think we want to verify that it's correct.